esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -3703,12 +3669,14 @@ int CWsWuFileHelper::zipAFolder(const char* folder, bool gzip, const char* zipFi
 }
 
 void CWsWuFileHelper::createWUZAPFile(IEspContext& context, Owned<IConstWorkUnit>& cwu, CWsWuZAPInfoReq& request,
```

Comment:
>Owned<IConstWorkUnit>& cwu

Not new, but shouldn't be passing Owned<...> references around in general.
Method should take underlying type (e.g. IConstWorkUnit *). You can pass in the Owned<...> object and it will automatically be deferenced to the underlying type.

Reply:
Will switch to a pointer.